integration_test/ops_agent_test.go

```diff
@@ -3095,7 +3095,7 @@ func TestLoggingAgentCrashRestart(t *testing.T) {
 	})
 }
 
-func TestLoggingFluentbitSelfLogs(t *testing.T) {
+func TestLoggingSelfLogs(t *testing.T) {
```

Comment:
Do we want to make these separate tests? More of a question for @martijnvans, I guess…